src/cdk/v2/destinations/optimizely_fullstack/procWorkflow.yaml

```diff
@@ -59,7 +61,9 @@ steps:
     description: Fetch the data file from the data file url
     template: |
       const dataFileUrl = .destination.Config.dataFileUrl;
-      const rawResponse = await $.handleHttpRequest("get", dataFileUrl);
+      const urlPath = $.getRelativePathFromURL(dataFileUrl)
```

Comment:
this is good but it is a chance to make mistakes like people copy paste and URL might contain dynamic parts, I prefer hard coded endpoint paths

Reply:
I agree why you say hardcoded paths are needed. But in this case, I don't even know how does the URL path look like. Hence had to adopt this.

Reply:
got it, it is coming from the config